File: level 012/homework/homework.py
print("----- 1 -----")

for i in range(21):
    print(i)

print("----- 2 -----")

num = int(input("enter number:"))

if num % 2 == 0:
    print("number is even")
else:
    print("number is odd")

print("----- 3 -----")

for num in range(21):
    if num % 2 == 0:
        print(num)

print("----- 4 -----")

for i in range(50, 101):
    num = i + (i +1)
    print(num)

print("----- 4 -----")

for num in range(1, 51):
    if num % 5 == 0:
        print(num)

print("----- 5 -----")

for i in range(5, 11):
    num = i * (i + 1)
    print(num)

File: level 086/homework/task 05/homework.js
const h1 = document.querySelector("h1");

const N = prompt("Enter number of numbers:");
let numbers = [];
for(let i = 0; i < N; i++){
    const num = prompt(`Enter number #${i + 1}`);
    numbers.push(num);
}
let first = numbers[0];
numbers[0] = numbers[N - 1];
numbers[N - 1] = first;

h1.textContent = numbers;